components/providers/provider-detail-client.tsx

```
@@ -6,12 +6,15 @@ import Link from "next/link";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
 import { getFallbackImageUrl } from "@/lib/image-utils";
-import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogFooter, DialogTrigger } from "@/components/ui/dialog";
-import { Label } from "@/components/ui/label";
-import { Input } from "@/components/ui/input";
-import { Textarea } from "@/components/ui/textarea";
 import { Category } from "@/lib/api/providers";
+import { ProviderLocationMap } from "@/components/maps/provider-location-map";
+import { ProviderLocationDetails } from "@/components/providers/provider-location-details";
+import { ProviderSchemaMarkup } from "@/components/providers/provider-schema-markup";
+import { ProviderContactForm } from "@/components/providers/provider-contact-form";
+import { MapPin, Mail, Calendar, Phone } from "lucide-react";
```

ESLint

[error] 'Phone' is defined but never used.

(@typescript-eslint/no-unused-vars)